src/NzbDrone.Core/Update/UpdateCheckService.cs

```diff
@@ -30,13 +31,13 @@ public class CheckUpdateService : ICheckUpdateService
 
         public UpdatePackage AvailableUpdate()
         {
+            var latestAvailable = _updatePackageProvider.GetLatestUpdate(_configFileProvider.Branch, BuildInfo.Version);
+
             if (OsInfo.IsNotWindows && !_configFileProvider.UpdateAutomatically)
```

Comment:
I was looking at this the other day and thought it was weird, this means people without auto update on don't get counted, doesn't it?